packages/astro/src/default/components/Header.astro

```diff
@@ -14,8 +14,9 @@ const LOGO_EXTENSIONS = [
 
 let logo;
 
-for (const logoFilename of LOGO_EXTENSIONS) {
-  const exists = fs.existsSync(path.join('public', `logo.${logoFilename}`));
+for (const logoExt of LOGO_EXTENSIONS) {
+  const logoFilename = `logo.${logoExt}`;
+  const exists = fs.existsSync(path.join('public', logoFilename));
```

Comment:
Woops, that's my fault 🤦 . I ran the project and the only thing I looked at was the warning...

Reply:
No worries! :raised_hands: